src/chrome/content/rules/Gametracker.com.xml

```diff
@@ -1,11 +1,7 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://image.www.gametracker.com/ => https://image.www.gametracker.com/: (60, 'SSL certificate problem: certificate has expired')
--->
-<ruleset name="Gametracker.com (partial)" default_off='failed ruleset test'>
+<ruleset name="Gametracker.com (partial)">
   <target host="www.gametracker.com"/>
   <target host="gametracker.com"/>
-  <target host="image.www.gametracker.com"/>
+  <target host="image.gametracker.com"/>
   <rule from="^http://(?:www\.)?gametracker\.com/" to="https://www.gametracker.com/"/>
-  <rule from="^http://image\.www\.gametracker\.com/" to="https://image.www.gametracker.com/"/>
+  <rule from="^http://image\.gametracker\.com/" to="https://image.gametracker.com/"/>
```

Comment:
Our style guide recommends to avoid non-capturing groups.
You can just replace them with a simple `<rule from="^http:" to="https:" />` rule.